src/main/java/com/khutircraftubackend/services/registration/RegistrationSellerService.java

```diff
@@ -0,0 +1,72 @@
+package com.khutircraftubackend.services.registration;
+
+import com.khutircraftubackend.dto.SellerDTO;
+import com.khutircraftubackend.mapper.SellerMapper;
+import com.khutircraftubackend.models.SellerEntity;
+import com.khutircraftubackend.repositories.SellerRepository;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class RegistrationSellerService {
```

Comment:
this could be just SellerService
as of now, no need to split everything that much

Reply:
I undestand.